### PR TITLE
doc: minor fixups

### PR DESCRIPTION
**Pull request by eadwu**

### CODE REVIEW

File: doc/readme/EXAMPLES.md [hunk 1]
```diff
@@ -94,14 +94,14 @@ You can render it with
...
+In case the `cookbook` target is still missing then inspect the following.
 
 Find out if value of `SPHINX_EXECUTABLE` is set by searching for it in CMakeCache.txt which is present in the build directory. If it's not set, then it is the `sphinx-build` that has not been found by the cmake. In this case re-run the cmake script with a explicit path to `sphinx-build` file. The file can be located by using the command `locate sphinx-build`. Once `sphinx-build` has been located, use the `-DSPHINX_EXECUTABLE="<path/to/sphinx-build>"` cmake flag to specify the `sphinx-build` location. In other words re-run cmake with the following options:
 
     cmake -DSPHINX_EXECUTABLE="<path/to/sphinx-build>" -DBUILD_META_EXAMPLES=ON [other cmake options] ..
 
 After the cookbook has been rendered, you can view it for example running
 
-    python -m SimpleHTTPServer
+    python -m http.server
```
Comment (gf712) [inline doc/readme/EXAMPLES.md:109]: you should keep both, even though python2 is not supported anymore, some ppl might still use it as their default

Comment (eadwu) [reply doc/readme/EXAMPLES.md:109]: Readded.

File: doc/readme/EXAMPLES.md [hunk 2]
```diff
@@ -94,14 +94,18 @@ You can render it with
...
 
-In case the `cookbook` target is still missing then inspect the following. 
+In case the `cookbook` target is still missing then inspect the following.
 
 Find out if value of `SPHINX_EXECUTABLE` is set by searching for it in CMakeCache.txt which is present in the build directory. If it's not set, then it is the `sphinx-build` that has not been found by the cmake. In this case re-run the cmake script with a explicit path to `sphinx-build` file. The file can be located by using the command `locate sphinx-build`. Once `sphinx-build` has been located, use the `-DSPHINX_EXECUTABLE="<path/to/sphinx-build>"` cmake flag to specify the `sphinx-build` location. In other words re-run cmake with the following options:
 
     cmake -DSPHINX_EXECUTABLE="<path/to/sphinx-build>" -DBUILD_META_EXAMPLES=ON [other cmake options] ..
 
-After the cookbook has been rendered, you can view it for example running
+After the cookbook has been rendered, you can view it for example with Python 2 running
```
Comment (karlnapf) [inline doc/readme/EXAMPLES.md:103]: I would just remove the python2 (not even supported by shogun anymore iirc?)

File: doc/readme/INTERFACES.md [hunk 3]
```diff
@@ -30,9 +30,9 @@ Make sure you read up on how to compile C/C++ code.
 Compilation requires the Shogun headers path, i.e. the path where for example `shogun/base/init.h` is located.
 This is either in `path/to/src/shogun/` or in `path/to/shogun-install/include/shogun/` and is specified  via the `-I` flag.
 Linking requires the `-lshogun` flag, which either needs the `LD_LIBRARY_PATH` or `DYLD_LIBRARY_PATH` set up as described above, or preferably passed via the `-L` flag.
-Compiling and linking code with `gcc` works as
+Compiling and linking code with `g++` works as
 
-    gcc path/to/native_example.cpp -o native_example -I/path/to/headers -lshogun -L/path/to/libshogun.*
+    g++ path/to/native_example.cpp -o native_example -std=c++17 -I/path/to/headers -lshogun -L/path/to/libshogun.*
```
Comment (karlnapf) [inline doc/readme/INTERFACES.md:35]: good catch!

File: doc/readme/EXAMPLES.md (hunk 2 above)
Comment (eadwu) [reply doc/readme/EXAMPLES.md:103]: It was recommended to keep it, https://github.com/shogun-toolbox/shogun/pull/4911#discussion_r391179011.

File: doc/readme/EXAMPLES.md (hunk 1 above)
Comment (karlnapf) [reply doc/readme/EXAMPLES.md:109]: ah ok this didnt show up in my github view. never mind then

**Comment by eadwu** [comment]
Rebased to resolve conflict.

**Pull request merged**